astrosat_users/templates/astrosat_users/email/invitation_subject.txt

```diff
@@ -0,0 +1,11 @@
+{# used for notifying a user that they have been added to a customer #}
+
+{% load i18n %}
+
+{% autoescape off %}
+
+{% blocktrans %}
+  Welcome to {{customer}} - Please sign in
```

Comment:
I would say this should say something like `Please complete invitation`, rather than `Please sign in`, but I doubt that was your decision.

Reply:
This was decided by the design team.  Actually, it's supposed to say "Welcome to ORBIS - Please sign in".  But I'm going to override these templates in **orbis**.

Reply:
Okay, makes sense, but the point still stands, the user has not yet completed the invitation, so it really should be explicit, too often users get confused because of poorly defined instructions in my experience. Still, don't want to delay your work, so I'll resolve this conversation, with reservations.